h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -325,7 +326,7 @@ function SidebarInjector(chromeTabs, dependencies) {
     var configCode =
       'var hypothesisConfig = "' + configStr + '";\n' +
       '(' + addJSONScriptTagFn.toString() + ')' +
-      '("js-hypothesis-config", hypothesisConfig)';
+      '("js-hypothesis-config", hypothesisConfig);\n';
```

Comment:
The trailing semi-colon and new line is not essential here. I just added it for consistency.